v1/Makefile

```diff
@@ -0,0 +1,46 @@
+.PHONY: major minor patch release build test push bump-git bump
+
+VERSION=$(shell cat version)
+IMAGE_NAME=sysdiglabs/secure-inline-scan
+INLINE_SCAN_SCRIPT=inline_scan.sh
+
+test: shellcheck
+
+major:
+	$(MAKE) release INCREMENT='major'
+
+minor:
+	$(MAKE) release INCREMENT='minor'
+
+patch:
+	$(MAKE) release INCREMENT='patch'
+
+release:
+	$(MAKE) bump INCREMENT=$(INCREMENT)
+	$(MAKE) push
+
+build:
+	docker build -t $(IMAGE_NAME):$(VERSION) -f Dockerfile . --no-cache
```

Comment:
I think this is not gonna work since now the script file is located one directory upper

Reply:
Thanks, now it should be fixed.